Sources/Networking/Misc/Counter.swift

```diff
@@ -8,7 +8,8 @@
 import Foundation
 
 /// A thread safe wrapper for count dictionary.
-actor Counter {
+@NetworkingActor
+final class Counter {
```

Comment:
I don't think counter needs to be tied to `@NetworkingActor`, it can simple be an actor 😄